src/zepben/evolve/model/cim/iec61970/base/wires/create_wires_components.py

```diff
@@ -0,0 +1,484 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from typing import List
+
+from zepben.evolve import *
+from zepben.evolve.model.common_two_way_connections import add_conducting_equipment_connection, \
+    add_power_electronics_connection_connection
+
+
+def create_ac_line_segment(mrid: str = None, name: str = '', description: str = "", names: List[Name] = None, location: Location = None,
```

Comment:
Names being a pain again. I think you need an `IdentifiedObject` to create a `Name`. If so, you can't pass any valid names into these functions.

Reply:
AFAIK identified_object is optional on name creation, all you need is name: str and type: NameType.